PROFILES.md

```diff
@@ -0,0 +1,72 @@
+# Interface of the Profiles.json
+
+```ts
+declare module namespace {
+
+    export interface Profiles {
+        globals: Globals;
+        profiles: Profile[];
+        schemes: Scheme[];
+    }
+
+    export interface Keybinding {
+        command: string;
+        keys: string[];
+    }
+
+    export interface Globals {
+        alwaysShowTabs: boolean;
+        defaultProfile: string;
+        initialCols: number;
+        initialRows: number;
+        keybindings: Keybinding[];
+        requestedTheme: string;
+        showTabsInTitlebar: boolean;
+        showTerminalTitleInTitlebar: boolean;
+    }
+
+    export class Profile {

```

Comment:
I'm fairly sure in the time this PR has been open, we've added a `tabTitle` member. @cinnamon-msft Do we need to make another PR to sync up the settings docs? If we do, then I'm fine excluding it here.

Reply:
@zadjii-msft @cinnamon-msft Hi,

shall i update this PR or create another one after this?

Reply:
I would update this with `tabTitle`, then I'm happy to approve it 😊